Cargo.toml

```diff
@@ -197,6 +197,7 @@ cc = { version = "1.0.83", default-features = false }
 default = ["alloc", "dev_urandom_fallback"]
 alloc = []
 dev_urandom_fallback = []
+less-safe-getrandom-custom = ["getrandom/custom"]
```

Comment:
I don't think that *ring* needs to activate the `getrandom/custom` feature. The project will have to actually define the custom implementation in some crate, and *that* crate can activate the feature. Or they could decide to implement the `rdrand` feature instead.

Note that `getrandom`'s "custom" feature also doesn't work in the expected way on targets that have RDRAND; see https://github.com/rust-random/getrandom/issues/346.

Accordingly, let's rename this to `less-safe-getrandom-custom-or-rdrand` and use `[]` as the dependency.